app/ui/src/app/integration/import-export/integration-import-export.component.html

```diff
@@ -18,26 +18,13 @@
         </ol>
       </div>
       <div class="toolbar-pf-action-right">
-        <button class="btn btn-default"
-                (click)="showCancelModal()">
-          Cancel
+        <button class="btn btn-primary"
+                (click)="redirectBack()">
+          Done
```

Comment:
Shouldn't we have i18n for this?

Reply:
I'll get it in the next pass through :-)